assets/src/design-system/theme/constants/zIndex.js

```diff
@@ -15,5 +15,9 @@
  */
 
 export const Z_INDEX = {
+  BOTTOM: 0,
+  DEFAULT: 1,
+  MIDDLE: 2,
+  TOP: 3,
```

Comment:
I feel a little hesitant on this naming schema because of existing use in the apps of zIndex constants.  The dashboard's got its z_index constants at the top level (which should go away mostly) and the editor has _many_ layers that will probably retain its z_index structure. Ideally we don't have to manipulate the z_index much in the design system, but for the sake of readability and knowing which one goes where should we need a "TOP_TOP" 😆  could you update these to be specific?

![Screen Shot 2021-02-03 at 9 17 57 AM](https://user-images.githubusercontent.com/10720454/106775848-be14cf00-6600-11eb-86fb-7bdada497af6.png)
![Screen Shot 2021-02-03 at 9 17 16 AM](https://user-images.githubusercontent.com/10720454/106775852-bead6580-6600-11eb-9b4c-e9d241e1ae2f.png)

Reply:
After chatting with Brittany and Max we decided to remove the global `Z_INDEX` constant since z-index is relative to siblings and not ancetors/descendants.